rs/messaging/src/message_routing.rs

```diff
@@ -270,8 +270,8 @@ pub(crate) struct MessageRoutingMetrics {
     pub(crate) remote_certified_heights: IntGaugeVec,
     /// Batch processing phase durations, by phase.
     pub(crate) process_batch_phase_duration: HistogramVec,
-    /// Number of timed out requests.
-    pub(crate) timed_out_requests_total: IntCounter,
+    /// Number of timed out messages.
+    pub(crate) timed_out_messages_total: IntCounter,
     /// Height at which the subnet last split (if during the lifetime of this
```

Comment:
Yes, for best-effort calls we also time out responses in output queues (because presumably the caller's subnet has already generated a timeout response locally, so shipping the response over XNet streams is likely a waste of resources).

The logic is all there, we just don't have any best-effort messages yet. So once this change is merged, we will still only be timing out guaranteed response requests in output queues, for the time being.